DevelopedCalculator.js

```diff
@@ -19,17 +19,19 @@ class DevelopedCalculator extends Calculator {
         this.historyElement.textContent = this.history.join(",")
     };
     onOperatorButtonClick(event) {
-            // calling parent class's onOperatorButtonClick method
-            super.onOperatorButtonClick(event);
-            // pushing current result and the operator to the history array
-            this.history.push(this.currentResult, this.operator);
-            // calling gettingHistoryArray to update the history element after clicking any operator button 
-            this.gettingHistoryArray();
-            // this.history.push(this.currentResult, this.operator);
-        }
-        //onClearButtonClick() {
-        //    super.onClearButtonClick()
-        // }
+        // calling parent class's onOperatorButtonClick method
+        super.onOperatorButtonClick(event);
+        // pushing current result and the operator to the history array
+        this.history.push(this.currentResult, this.operator);
+        // calling gettingHistoryArray to update the history element after clicking any operator button 
+        this.gettingHistoryArray();
+        // this.history.push(this.currentResult, this.operator);
+    }
+    onClearButtonClick() {
+        super.onClearButtonClick()
+        this.history = [""]
+        this.historyElement.textContent = this.history
```

Comment:
what will happen if you call `this.gettingHistoryArray();` instad of `this.historyElement.textContent = this.history`